manifest_staging/charts/csi-secrets-store-provider-azure/arc-values.yaml

```diff
@@ -0,0 +1,195 @@
+    rollingUpdate:
+      maxUnavailable: 1
+  privileged: false
+  # If provided, the userAgent string will be appended to the
+  # AKV provider user agents for all adal and keyvault requests.
+  customUserAgent: ""
+  healthzPort: 8989
+  healthzPath: "/healthz"
+  healthzTimeout: "5s"
+  volumes: []
+  volumeMounts: []
+  kubeletRootDir: /var/lib/kubelet
+  providersDir: /var/run/secrets-store-csi-providers
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: type
+            operator: NotIn
+            values:
+            - virtual-kubelet
+
+windows:
+  image:
+    repository: mcr.microsoft.com/oss/azure/secrets-store/provider-azure
+    tag: v1.1.0
+    pullPolicy: IfNotPresent
+  nodeSelector: {}
+  tolerations: []
+  enabled: false
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+    limits:
+      cpu: 100m
+      memory: 200Mi
+  podLabels: {}
+  podAnnotations: {}
+  priorityClassName: ""
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  # If provided, the userAgent string will be appended to the
+  # AKV provider user agents for all adal and keyvault requests.
+  customUserAgent: ""
+  healthzPort: 8989
+  healthzPath: "/healthz"
+  healthzTimeout: "5s"
+  volumes: []
+  volumeMounts: []
+  kubeletRootDir: C:\var\lib\kubelet
+  providersDir: C:\k\secrets-store-csi-providers
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: type
+            operator: NotIn
+            values:
+            - virtual-kubelet
+
+## Configuration values for the secrets-store-csi-driver dependency.
+## ref: https://github.com/kubernetes-sigs/secrets-store-csi-driver/tree/master/charts/secrets-store-csi-driver/README.md
+##
+secrets-store-csi-driver:
+  logFormatJSON: true
+  install: true
+  # By default helm will append the chart release name to the dependent chart names.
+  # Explicitly setting the fullnameOverride will override this behavior.
+  fullnameOverride: secrets-store-csi-driver
+  linux:
+    enabled: true
+    kubeletRootDir: /var/lib/kubelet
+    metricsAddr: ":8080"
+    image:
+      repository: mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver
+      tag: v1.1.2
+      pullPolicy: IfNotPresent
+    registrarImage:
+      repository: mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar
+      tag: v2.5.0
+      pullPolicy: IfNotPresent
+    livenessProbeImage:
+      repository: mcr.microsoft.com/oss/kubernetes-csi/livenessprobe
+      tag: v2.6.0
+      pullPolicy: IfNotPresent
+    crds:
+      image:
+        repository: mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver-crds
+        tag: v1.1.2
+        pullPolicy: IfNotPresent
+    podAnnotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "8080"
```

Comment:
Should we also set pod annotations for the provider pods?

Reply:
Since template for provider pods is available in the helm chart, I directly added it there 

https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/833/files#diff-61da5351f8d0f9f5e31cdda817f47cf0c629827a5d9721035d13acbde069e80eR25-R28